uitools/cpp/Esri/ArcGISRuntime/Toolkit/SearchViewController.cpp

```diff
@@ -188,7 +212,13 @@ namespace Toolkit {
 
           if (sceneView->isNavigating())
           {
-            setIsEligableForRequery(true);
+            auto widthDiff = queryArea().extent().width() - lastSearchArea().extent().width();
+            auto heightDiff = queryArea().extent().width() - lastSearchArea().extent().width();
+
+            auto widthThreshold = lastSearchArea().extent().width() * 0.25;
+            auto heightThreshold = lastSearchArea().extent().height() * 0.25;
+            if (widthDiff > widthThreshold || heightDiff > heightThreshold)
```

Comment:
❓  The `sceneView` section is missing you `Check center difference` checks as seen in `MapView`. Is this intended?

🔧  There's enough code repetition here it would be worth centralising the logic instead of copy+pasting. Will help future maintainers.